### PR TITLE
Fix:  ast-offset fix

### LINKED ISSUE #1: span start/end keeps increasing on https://swc.rs/playground
How to reproduce:

1. In "Output" pick "AST" view.
2. Type `1` in "Input" view.
3. Keep adding spaces after `1`.
4. Notice how span start/end properties keep going up.

Expected: don't artificially increase indexes on start/stop properties.

### PR DESCRIPTION
**Pull request by IWANABETHATGUY**
fix #1

### CODE REVIEW

**Comment by IWANABETHATGUY** [comment]
hi @g-plane , the last pr commit history is pretty messy, so I reopen here, this pull request use no `any` and only one cast https://github.com/swc-project/swc-playground/pull/9/files#diff-39cc19ac7b95ec6821d23f87ff94218f89b639e42d335fa5d17e09cb2cf404fcR74 which is necessary

**Review by g-plane — CHANGES_REQUESTED** [review]
1. Identifiers must follow camel case convention.
2. Don't use `any`.
3. Those imported and not used as values must be marked as "type-only import".

**Comment by IWANABETHATGUY** [comment]
> 1. Identifiers must follow camel case convention.
> 2. Don't use `any`.
> 3. Those imported and not used as values must be marked as "type-only import".

sorry about that , recently i write rust a lot, hhhh.

File: src/components/Workspace.tsx [hunk 1]
```diff
@@ -92,3 +99,30 @@ export default function Workspace() {
...
   )
 }
+
+function adjustOffsetOfAst(obj: unknown, startOffset: number) {
+  if (Array.isArray(obj)) {
+    obj.forEach((item) => adjustOffsetOfAst(item, startOffset))
+  } else if (isRecord(obj)) {
+    const keys = Object.keys(obj)
+
+    keys.forEach((key) => {
```
Comment (g-plane) [inline src/components/Workspace.tsx]: Personally using `Object.entries` could be better.

**Review by g-plane — APPROVED** [review]
LGTM, thanks!

**Pull request merged**